BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/Board.java

```diff
@@ -0,0 +1,51 @@
+package pl.jakubowskiprzemyslaw.tajgertim.models.board;
```

Comment:
Your packages are not well aligned:

      1 pl.jakubowskiprzemyslaw.REST.configuration;
      1 pl.jakubowskiprzemyslaw.REST.listeners;
      1 pl.jakubowskiprzemyslaw.REST.models;
      1 pl.jakubowskiprzemyslaw.tajgertim.models.player;
      1 pl.jakubowskiprzemyslaw.tajgertim.models.playeraction;
      1 pl.jakubowskiprzemyslaw.tajgertim.queues;
      2 pl.jakubowskiprzemyslaw.REST;
      2 pl.jakubowskiprzemyslaw.tajgertim.models.configuration;
      2 pl.jakubowskiprzemyslaw.tajgertim.models.round;
      3 pl.jakubowskiprzemyslaw.REST.services;
      3 pl.jakubowskiprzemyslaw.tajgertim.models.coordinates;
      3 pl.jakubowskiprzemyslaw.tajgertim.models.playeraction.action;
      3 pl.jakubowskiprzemyslaw.tajgertim.models.shoot;
      3 pl.jakubowskiprzemyslaw.tajgertim.models.view;
      3 pl.jakubowskiprzemyslaw.tajgertim.services;
      4 pl.jakubowskiprzemyslaw.tajgertim;
      4 pl.jakubowskiprzemyslaw.tajgertim.configuration;
      5 pl.jakubowskiprzemyslaw.tajgertim.service;
      6 pl.jakubowskiprzemyslaw.REST.controllers;
      6 pl.jakubowskiprzemyslaw.tajgertim.models.board;
      7 pl.jakubowskiprzemyslaw.tajgertim.models;

1. There is a number of packages which do not have `tajgertim`, consider changing it (as a separate task, not in this PR)
2. Take a look at packages above with just one class. Consider if they are really needed and consider what public API you have there. I'm not saying "remove them", I'm saying - take a look.
3. I've taken a look at your objects in model packages, it's all over the place, I can't really say what you have when I look at `model`s. Perhaps I've spent too little time looking (6 minutes) but PR is quite large.

Next general remarks when it comes to packages: you're writing microservices, why would you need so many public APIs? Why aren't package APIs enough?

Next general remarks: number of your exceptions extends Throwable, why is that? You lose the general catch-all via `catch (Exception e)`. Is there a reason for it?

Reply:
Will be done in next sprint. 
Issue is tracked under #57